cmd2.py

```diff
@@ -748,7 +748,9 @@ def replace_with_file_contents(fname):
     :return: str - contents of file "fname"
     """
     try:
-        with open(os.path.expanduser(fname[0])) as source_file:
+        # Any outer quotes are not part of the filename
```

Comment:
Nice little fix.  It would be undesirable to create a file name with quotes in it.  

The new behavior is much better.